drivers/flash/jesd216.c

```diff
@@ -286,3 +286,44 @@ int jesd216_bfp_decode_dw14(const struct jesd216_param_header *php,
 
 	return 0;
 }
+
+int jesd216_bfp_decode_dw15(const struct jesd216_param_header *php,
+			    const struct jesd216_bfp *bfp,
+			    struct jesd216_bfp_dw15 *res)
+{
+	/* DW15 introduced in JESD216A */
+	if (php->len_dw < 15) {
+		return -ENOTSUP;
+	}
+
+	uint32_t dw15 = sys_le32_to_cpu(bfp->dw10[5]);
+
+	res->hold_reset_disable = (dw15 & BIT(23)) != 0U;
+	res->qer = (dw15 >> 20) & 0x07;
+	res->entry_044 = (dw15 >> 16) & 0x0F;
+	res->exit_044 = (dw15 >> 10) & 0x3F;
+	res->support_044 = (dw15 & BIT(9)) != 0U;
+	res->enable_444 = (dw15 >> 4) & 0x0F;
```

Comment:
The filed is 5 bit long, shouldn't the mask be 0x1F?

Reply:
Quite right; fixed; thanks.